app/src/test/kotlin/com/wire/android/core/network/ApiServiceTest.kt

```diff
@@ -1,16 +1,8 @@
 package com.wire.android.core.network
 
 import com.wire.android.UnitTest
-import com.wire.android.core.exception.BadRequest
-import com.wire.android.core.exception.EmptyResponseBody
-import com.wire.android.core.exception.Failure
-import com.wire.android.core.exception.Forbidden
-import com.wire.android.core.exception.InternalServerError
-import com.wire.android.core.exception.NetworkConnection
-import com.wire.android.core.exception.NotFound
-import com.wire.android.core.exception.ServerError
-import com.wire.android.core.exception.TooManyRequests
-import com.wire.android.core.exception.Unauthorized
+import com.wire.android.core.exception.*
```

Comment:
As I remember wildcard import is not allowed

Reply:
Forgot to bring IDE settings to new setup 😅 

Done :)